docs/source/azure/samplepolicies/nsg.rst

```diff
@@ -0,0 +1,76 @@
+Network Security Groups
+=======================
+
+Available Types:
+----------------
+    - ingress - Inbound Security Rules
+    - egress - Outbound Security Rules
+
+Available Filters:
```

Comment:
`ec2.rst` is a good example of how to set these up.  It looks like it depends on some generated stuff too which we need to enable for Azure.